app/controllers/users_controller.php

```diff
@@ -973,6 +973,7 @@ function delete($id, $courseId = null)
 
         // Ensure that the id is valid
         if (!is_numeric($id)) {
+            $this->log('MMMMMM-users_controller::'.$id);
```

Comment:
Is this for temporary debug? If yes, could you please remove it? or make the log message more meaningful if you want to keep it.